tests/brun/lsh-c.txt

```diff
@@ -1,3 +1,2 @@
 brun -c '(lsh (q . 0x00000000000000000000000000000000000000000000000000000000000000000007) (q . 0x000000000000000000000000000000000000000000000000000000000000000000001))'
-cost = 76
-14
+FAIL: lsh requires int32 args (with no leading zeros) 0x0000000000000000000000000000000000000000000000000000000000000000000001
```

Comment:
Hmm, now that these fail because of arg size, we need new tests on the boundary.

Reply:
good point, I'll add those (in a new PR)